course/test_exportcourse.py

```diff
@@ -40,9 +41,22 @@ def test_argument_type(self):
                 stdout=StringIO(),
                 verbosity=3)
 
+
+class audioIdTest(TestCase):
+    databases = '__all__'
```

Comment:
Database and fixtures not needed here, because the tests don't use it. Database makes things slower, so let's skip it when it's not necessary

Reply:
(only true for AudioIdTest)

Reply:
My bad,thought i deleted it